### PR TITLE
CI: optimize buildah build x2

### PR DESCRIPTION
**Pull request by TriplEight**
Fix the issue when one wants two tags has to build the image twice.

### CODE REVIEW

**Comment by TriplEight** [comment]
(debug build was built successfully, I didn't expect it to publish)
Can be merged.

File: .github/workflows/publish-release-container.yml
Comment (emostov) [inline .github/workflows/publish-release-container.yml:20]: Just to understand better, is this building two images simultaneously or just one image with two tags?

**Review by emostov — APPROVED** [review]
Seems reasonable to me

File: .github/workflows/publish-release-container.yml
Comment (TriplEight) [reply .github/workflows/publish-release-container.yml:20]: The previous version did build two images simultaneously, this now just adds tags.

**Comment by emostov** [comment]
Any chance we can get a second review from @s3krit or @rcny ?

**Review by rcny — APPROVED** [review]

**Pull request merged**
